react/features/base/lib-jitsi-meet/actionTypes.js

```diff
@@ -27,6 +27,16 @@ export const LIB_DID_INIT = Symbol('LIB_DID_INIT');
  */
 export const LIB_INIT_ERROR = Symbol('LIB_INIT_ERROR');
 
+/**
+ * Action to dispatch the promise returned by JitsiMeetJS.init.
+ *
+ * {
+ *     type: LIB_INIT_ERROR,
```

Comment:
Please review this comment for accuracy and the review the spelling of LIB_INIT_POMISE_CREATED.